driver/clirr-ignored-differences.xml

```diff
@@ -122,4 +122,150 @@
         <method>org.reactivestreams.Publisher isOpen()</method>
     </difference>
 
+    <difference>
```

Comment:
Doesn't clirr support "this is a big change, we bump major anyway, please leave me alone?"

Reply:
I might explore later. I find it useful for keeping track of these updates and it serves as a changelog at times.